lib/manager/jsonnet-bundler/extract.ts

```diff
@@ -0,0 +1,62 @@
+import { GitTagsDatasource } from '../../datasource/git-tags';
+import { logger } from '../../logger';
+import { regEx } from '../../util/regex';
+import { PackageDependency, PackageFile } from '../types';
+import { Dependency, JsonnetFile } from './types';
+
+const gitUrl = regEx(
+  /(ssh:\/\/git@|https:\/\/)([\w.]+)\/([\w:/\-~]*)\/(?<depName>[\w:/-]+)(\.git)?/
+);
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string
+): PackageFile | null {
+  logger.trace(`jsonnet-bundler.extractPackageFile(${packageFile})`);
+  logger.trace({ content });
+
+  if (packageFile.match(/vendor\//)) {
+    return null;
+  }
+
+  const deps: PackageDependency[] = [];
+  let jsonnetFile: JsonnetFile;
+  try {
+    jsonnetFile = JSON.parse(content);
+  } catch (err) {
+    logger.debug({ packageFile }, 'Invalid JSON');
+    return null;
+  }
+
+  for (const dependency of jsonnetFile.dependencies ?? []) {
+    const dep = extractDependency(dependency);
+    if (dep) {
+      deps.push(dep);
+    }
+  }
+
+  if (!deps.length) {
+    return null;
+  }
+
+  return { deps };
+}
+
+export function extractDependency(
```

Comment:
Don't export if  there is no public use.

Reply:
```suggestion
function extractDependency(
```